app/uk/gov/hmrc/gform/eval/EvaluationResults.scala

```diff
@@ -337,6 +353,12 @@ case class EvaluationResults(
             } yield result).getOrElse("")
           )
         )
+      case Size(formComponentId, index) =>
+        nonEmpty(
+          StringResult(
+            evalSize(formComponentId, recData, index).stringRepresentation(typeInfo, evaluationContext.messages)
+          )
+        )
```

Comment:
This should be simply

```
evalSize(formComponentId, recData, index)
```

I don't see any reason for turning it into `StringResult`